crates/steel-doc/src/main.rs

```diff
@@ -53,7 +53,8 @@ fn main() {
 
                 match value {
                     steel::steel_vm::builtin::Documentation::Markdown(m) => {
-                        writeln!(&mut module_file, "### **{}**", name).unwrap();
+                        let escaped = name.replace("*", "\\*");
+                        writeln!(&mut module_file, "### **{}**", escaped).unwrap();
```

Comment:
In lieu of proper markdown generation, this prevents the `*` operator from appearing as `*****` :sweat_smile:

Reply:
Yes, that's the only one I noticed.